web/packages/api/src/environment.ts

```diff
@@ -10,11 +10,6 @@ export type Config = {
     SECONDARY_GOVERNANCE_CHANNEL_ID: string
     RELAYERS: Relayer[]
     PARACHAINS: { [paraId: string]: string }
-    SUBSCAN_API?: {
```

Comment:
why do all these config keys need to be in uppercase? These arent environment variables.